addon/plugins/variable-plugin/nodes.ts

```diff
@@ -58,7 +58,110 @@ export const getPNodeExtraAttributes = ({
   return {};
 };
 
-const emberNodeConfig: EmberNodeConfig = {
+export const parseAttributes = (node: HTMLElement): false | Attrs => {
```

Comment:
With the variable number being a separate node now both variables will have different parsing and while is good so simplify code in a function I don't it's something we can do here, or at least not like this.
The minimun and maximum value should be part only of the number variable for example

Reply:
@x-m-el I think this is a good approach for now, the merge conflicts will indeed be more minimal